### DISCUSSION TITLE
Disable AOT on iOS ?

### DISCUSSION THREAD

**Discussion by Symbai** [dotnet/maui#8844] [Q&A] [answered]
My app crashes only in release build when using AOT. For Android I just disabled AOT using `<RunAOTCompilation>false</RunAOTCompilation>`, how can I do the same for iOS?

**Comment by xtuzy** [comment, marked as answer]
Apple require AOT, https://docs.microsoft.com/en-us/xamarin/ios/internals/architecture#aot. you should find bug.

**Reply by Symbai** [reply]
> you should find bug

How? When the app only crashes in release and only on physical devices?

**Reply by xtuzy** [reply]
you can see device log, you can write log at some key method, another i advice you release a default app to compare

**Reply by Symbai** [reply]
It crashes right on start (after splashscreen). There is no code of mine executed. I posted everything I got as an issue here but no replies yet. How can I get the device log from the iphone?

**Reply by xtuzy** [reply]
if it don't run any code of you, you need try release a default project, avoid some setting in .csproj not correct. 

in visualstudio can get device log, at Tool/iOS/

**Reply by Symbai** [reply]
There is no Tool/iOS

**Reply by xtuzy** [reply]
![sharex-20220722103751](https://user-images.githubusercontent.com/17793881/180350754-3798483d-c849-4263-8bab-cd1733e65db3.png)
You also can google other method

**Reply by Symbai** [reply]
Ah on windows machine. That worked thank you. Unfortunately I am unable to understand what the log tells me but I've filled out an issue with it.
